sql/mozfun/norm/browser_version_info/udf.sql

```diff
@@ -0,0 +1,76 @@
+CREATE OR REPLACE FUNCTION norm.browser_version_info(version_string STRING)
+RETURNS STRUCT<
+  version STRING,
+  major_version NUMERIC,
+  minor_version NUMERIC,
+  is_major_release BOOLEAN
+> AS (
+  STRUCT(
+    version_string AS version,
+    norm.truncate_version(version_string, 'major') AS major_version,
+    norm.truncate_version(version_string, 'minor') AS minor_version,
+    ARRAY_LENGTH(SPLIT(version_string, '.')) = 2
+    AND ENDS_WITH(version_string, '.0') AS is_major_release
+  )
+);
+
+-- Tests
```

Comment:
I actually love this. Are these queries / "tests" executed as part of CI in any way prior to deployment?